device/cuda/src/clusterization/clusterization_algorithm.cu

```diff
@@ -0,0 +1,305 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Library include(s).
+#include "traccc/cuda/clusterization/clusterization_algorithm.hpp"
+
+// Project include(s)
+#include "traccc/clusterization/device/connect_components.hpp"
+#include "traccc/clusterization/device/count_cluster_cells.hpp"
+#include "traccc/clusterization/device/create_measurements.hpp"
+#include "traccc/clusterization/device/find_clusters.hpp"
+#include "traccc/clusterization/device/form_spacepoints.hpp"
+#include "traccc/device/get_prefix_sum.hpp"
+
+// Vecmem include(s).
+#include <vecmem/utils/copy.hpp>
+
+// System include(s).
+#include <algorithm>
+
+// Local include(s)
+#include "traccc/cuda/utils/definitions.hpp"
+
+namespace traccc::cuda {
+namespace kernels {
+
+__global__ void invoke_find_clusters_kernel(
```

Comment:
Why do you put `invoke_` to the kernel names, which is not the case for cuda seeding? - I know it's a bit pedantic though...

Reply:
I did so because it calls another kernel which does the actual work. Will follow similar thing done in seeding.